qrenderdoc/Windows/ShaderViewer.cpp

```diff
@@ -35,6 +35,7 @@
 #include <QSet>
 #include <QShortcut>
 #include <QToolTip>
+#include <QWidgetAction>
```

Comment:
Is there a reason for this added header? it doesn't seem to be used.

Reply:
Ah, something I forgot to remove...